packages/design-system/src/List/List.styles.tsx

```diff
@@ -0,0 +1,163 @@
+`;
+
+const Sizes = {
+  md: css`
+    --listitem-title-font-size: var(--ads-v2-font-size-4);
+    --listitem-bdescription-font-size: var(--ads-v2-font-size-2);
+    --listitem-idescription-font-size: var(--ads-v2-font-size-2);
+  `,
+  lg: css`
+    --listitem-title-font-size: var(--ads-v2-font-size-6);
+    --listitem-bdescription-font-size: var(--ads-v2-font-size-4);
+    --listitem-idescription-font-size: var(--ads-v2-font-size-4);
+  `,
+};
+
+export const StyledList = styled.div`
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  padding: var(--ads-v2-spaces-1);
+`;
+
+export const Wrapper = styled.div`
+  display: flex;
+  width: 100%;
+  align-items: center;
+  gap: 8px;
+  cursor: pointer;
+  box-sizing: border-box;
+  position: relative;
+`;
+
+export const StartIconWrapper = styled.div`
+  align-self: flex-start;
+`;
+
+export const TooltipTextWrapper = styled.div`
+  display: flex;
+  min-width: 0;
+`;
+
+export const ContentWrapper = styled.div`
+  display: flex;
+  gap: var(--ads-v2-spaces-3);
+`;
+
+export const ContentTextWrapper = styled.div`
+  display: flex;
+  gap: var(--ads-v2-spaces-3);
+  flex: 1;
+  min-width: 0;
+`;
+
+export const DescriptionWrapper = styled.div`
+  flex-direction: column;
+  min-width: 0;
+  gap: var(--ads-v2-spaces-3);
+  display: flex;
+`;
+
+export const InlineDescriptionWrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  min-width: 0;
+  gap: var(--ads-v2-spaces-3);
+  flex: 1;
+`;
+
+export const EndIconWrapper = styled.div`
+  position: absolute;
+  right: var(--ads-v2-spaces-3);
+`;
+
+export const StyledListItem = styled.div<{
+  size: ListSizes;
+  endIcon?: string;
+  isBlockDescription: boolean;
+}>`
+  ${Variables};
+
+  display: flex;
+  width: 100%;
+  align-items: center;
+  border-radius: var(--ads-v2-border-radius);
+  padding: var(--ads-v2-spaces-3);
+  box-sizing: border-box;
+  // 40px is the offset to make it look like the end icon is part of this div
+  ${(props) => !!props.endIcon && `padding: 8px 40px 8px 8px;`}}
+
+  ${({ size }) => Sizes[size]}
+
+  ${({ isBlockDescription, size }) =>
+    size === "lg" &&
+    isBlockDescription &&
+    `
+    ${DescriptionWrapper} {
+      padding-top: 3px
+    }
+  `}
+
+  & .${ListItemTextOverflowClassName} {
+    overflow: hidden;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+  }
+
+  & .${ListItemTitleClassName} {
+    font-size: var(--listitem-title-font-size);
+  }
+
+  & .${ListItemBDescClassName} {
+    -webkit-line-clamp: 2;
+    display: -webkit-box;
+    -webkit-box-orient: vertical;
+    text-overflow: initial;
+    white-space: initial;
+    font-size: var(--listitem-bdescription-font-size);
+    line-height: normal;
+  }
+
+  & .${ListItemIDescClassName} {
+    font-size: var(--listitem-idescription-font-size);
+  }
+
+  &:hover {
+    background-color: var(--ads-v2-color-bg-subtle);
+  }
+
+  &:active {
+    background-color: var(--ads-v2-color-bg-muted);
```

Comment:
use `--ads-v2-colors-content-surface-active-bg`

we should be using category tokens and not sdemantic